server/replica/character.py

```diff
@@ -0,0 +1,133 @@
+"""
+Character component
+"""
+
+from pyraknet.bitstream import c_bit, c_uint8, c_uint16, c_uint32, c_uint64, c_int32, c_int64
+from pyraknet.replicamanager import Replica
+
+from replica.component import Component
+from structs import WString
+
+
+class Character(Component):
+    """
+    Character replica
+    """
+    def __init__(self, vehicle=False, vehicle_id=0, level=False, level_num=1, hair_color=0, hair_style=0, shirt_color=0,
+                 pants_color=0, eyebrows=0, eyes=0, mouth=0, account_id=1, llog=0, lego_score=0, free_to_play=False,
+                 currency_collected=0, bricks_collected=0, smashables_smashed=0, quick_builds=0, enemies_smashed=0,
+                 rockets_used=0, missions_completed=0, pets_tamed=0, imagination_powerups=0, life_powerups=0, armor_powerups=0,
+                 distance_traveled=0, times_smashed=0, damage_taken=0, damage_healed=0, armor_repaired=0,
+                 imagination_restored=0, imagination_used=0, distance_driven=0, airborne_time_race_car=0,
+                 racing_imagination_powerups=0, racing_imagination_crates_smashed=0, race_car_boosts=0, race_car_wrecks=0,
+                 racing_smashables_smashed=0, races_finished=0, first_place_finishes=0, rocket=False, rocket_characters=0,
+                 rocket_modules=[''], glowing_head=False, guild=False, guild_id=0, guild_name=''):
+        super().__init__(**{k: v for k, v in locals().items() if k != 'self'})
+
+    def write_data1(self, stream):
```

Comment:
How about naming these `pre_creation` and `post_creation` or something? Or even create one function for each "part" as defined in the docs, then in part_4 have the first parameter be creation vs serialization or something